javascript/2-functions/hw-4/scripts.js

```diff
@@ -1 +1,95 @@
 // Your JS code is here
+
+
+// task 1
+function squareNumber(num) {
+  return
+    var san = num * num
+    console.log('The result of squaring the number ' + num + ' is ' + san)
+};
+var temp = Math.random() * 10 +10;
+squareNumber(temp.toFixed());
+
```

Comment:
here you have similar issue. Please see if your return will work here. and try to fix it. if not please feel free to reach me out.